### PR TITLE
Fix-x-button-not-working-properly

### LINKED ISSUE #148: Fix remove file button

### PR DESCRIPTION
**Pull request by MohamedBechirMejri**
Fixes #148

### CODE REVIEW

File: stories/Uploader.stories.tsx [hunk 1]
```diff
@@ -22,7 +25,10 @@ const Uploader = args => {
             multiple: true,
             limit: 5,
             onFileClick: file => void 0,
-            onFilesChange: files => void 0,
+            onFilesChange: files => {
+                console.log('files', files)
```
Comment (yaakoubiIheb) [inline stories/Uploader.stories.tsx:29]: I think the console log is not necessary.

**Comment by yaakoubiIheb** [comment]
GJ it's working fine just a little change.

**Review by yaakoubiIheb — CHANGES_REQUESTED** [review]

File: stories/Uploader.stories.tsx (hunk 1 above)
Comment (MohamedBechirMejri) [reply stories/Uploader.stories.tsx:29]: it's to test the callback in storybook so we should keep it

**Review by bassem97 — APPROVED** [review]
good job

**Pull request merged**